test/extended/testdata/templates/templateservicebroker_bind.yaml

```diff
@@ -47,7 +49,7 @@ objects:
           ports:
           - name: port
             port: 1234
-      - apiVersion: v1
+      - apiVersion: route.openshift.io/v1
```

Comment:
@liggitt I think this template covers all the variants you were concerned w/ now:

1) openshift api object by group (route)
2) openshift api object by legacy (service)
3) k8s api object by standard api (configmap)
4) k8s api object by group (CRD)

Reply:
> openshift api object by legacy (service)

service is a kube object

Reply:
yeah dunno what i was thinking.  updated w/ a second route object that uses the legacy api.